### ISSUE TITLE
FTP-Simple 

### ISSUE THREAD

**Issue by Mac1an** [humy2833/FTP-Simple#331]
Type: <b>Bug</b>

I am getting this error:

Command 'ftp-simple : Config - FTP connection setting' resulted in an error
Cannot read properties of undefined (reading '0')

Extension version: 0.7.6
VS Code version: Code 1.93.0 (Universal) (4849ca9bdf9666755eb463db297b69e5385090e3, 2024-09-04T13:02:38.431Z)
OS version: Darwin arm64 23.6.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 Max (10 x 2400)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|3, 3, 4|
|Memory (System)|32.00GB (1.77GB free)|
|Process Argv|--crash-reporter-id b136e211-e4fe-4698-9689-93cbb5c4da62|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805:30301674
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
a9j8j154:30646983
962ge761:30959799
pythongtdpath:30769146
welcomedialogc:30910334
pythonnoceb:30805159
asynctok:30898717
pythonmypyd1:30879173
h48ei257:31000450
pythontbext0:30879054
accentitlementst:30995554
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
dsvsc021:30996838
0ee40948:31013168
a69g1124:31058053
dvdeprecation:31068756
dwnewjupytercf:31046870
impr_priority:31102340
nativerepl2:31134655
refactort:31108082
pythonrstrctxt:31112756
flightc:31134773
wkspc-onlycs-t:31132770
wkspc-ranged-t:31125599
ei213698:31121563

```

</details>

**Comment by wirthjr** [comment]
I have the same error

**Comment by davidbicho** [comment]
I have error: Cannot create property 'autosave' on string 'c'

**Comment by andrescalfulaf** [comment]
I have the same error, and it is happening in Windows 11 since it does not happen on Mac.

**Comment by twedpaulosousa** [comment]
I have the same error

Windows 11 23H2
VSCode version 1.93
ftp-simple v0.7.6

![image](https://github.com/user-attachments/assets/bd26de20-56ae-478d-af74-46e5164b3369)

**Comment by andrescalfulaf** [comment]
I installed the SFTP:Natizyskunk v1.13.6 extension and somehow the ftp-simple error was fixed

**Comment by davidbicho** [comment]
Yes!!!! I uninstalled ftp-simple and then installed SFTP:Natizyskunk v1.13.6 and modified it's config file to my ftp-settings. Then I installed ftp-simple (latest version) and could log in, but I got an error from SFTP:Natizyskunk saying that there was some error in the config-file, so I edited it and changed from sftp to ftp-protocol and saved. This saved a configfile at my ftp-server (since this now was my working directory) and everything worked!

It feels like I am running in some kind of spaghetti solution and have no clear control of whats happening, but it works so I don't investigate further. When it works, dont change it :).

**Comment by SebasVergara** [comment]
> I installed the SFTP:Natizyskunk v1.13.6 extension and somehow the ftp-simple error was fixed

Life saver!

**Comment by Saykorans** [comment]
> I installed the SFTP:Natizyskunk v1.13.6 extension and somehow the ftp-simple error was fixed

BROTHER THANK YOU !! <3
